setup.py

```diff
@@ -18,6 +18,13 @@
 Intended Audience :: Information Technology
 License :: OSI Approved :: BSD License
 Programming Language :: Python
+Programming Language :: Python :: 2
```

Comment:
We also test on Travis on 2.6 and 3.3.

Reply:
Really??  2.6 EOL was five years ago and 3.3 was more than a year ago.

https://snarky.ca/stop-using-python-2-6 was written three years ago.

Reply:
Technically there's code that's only necessary in 2.4 and before, although I'm not even confident the rest of the codebase will work on 2.4 at all, and I'm certainly not going to propose we adjust our CI to enforce that it does.  It's just that no one's gotten around to removing it, and it doesn't particularly hurt anything as it is. Probably.

I personally wouldn't object to dropping support for older versions and Mr Cannon as usual makes some good points.